.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-{
-  "version": "1.0",
-  "tasks": [
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "config": {
-        "taskName": "Add needs triage and bug label to new issues",
```

Comment:
This task didn't get ported, presumably because it references `isPartOfProject` which is not supported.